SAML/env.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
I do not like having this `env.sh` file repeated all over the repo. If many scripts use the same file, we may extract it to the `root` folder, or so. But we can live with three repeated files so it is easier for the final user.